pandas_gbq/gbq.py

```diff
@@ -7,11 +7,10 @@
 import sys
 import os
 
-import numpy as np
-
 from distutils.version import StrictVersion
-from pandas import compat, DataFrame, concat
-from pandas.compat import lzip, bytes_to_str
+from pandas import compat, DataFrame, to_datetime, to_numeric
+from pandas.compat import bytes_to_str
+from google.cloud import bigquery
```

Comment:
the import from google.cloud should in theory be *inside* _check_google_client_version

Reply:
I think I see the point of doing the checks before the import of bigquery, but wouldn't putting the import of bigquery inside _check cause it to be scoped for only inside that function, and thus inaccessible to read_gbq? But feel free to correct and suggest how I should do it since I'm still a python novice, thank you!

Reply:
you can't have the imports like this as I stated above. Then the entire module will simply be an import error (and not a friendly one). The entire point is to provide a nice error message.

Reply:
Perhaps I'm misreading your comment? See my response about the imported module being no longer in scope if I import it in `_check_google_client_version()`. Doing so will give me a `NameError: global name 'bigquery' is not defined` error going forward. Perhaps I could import after `_test_google_api_imports()` inside the `read_gbq()` function? Would that resolve the issue for you?

Reply:
FYI, moved the import to inside GbqConnector since that's the only place it's needed (to generate the appropriate client object). read_gbq performs a `_test_google_api_imports()` at the very start, so if there are import issues, `_test_google_api_imports()` will provide an error message.